src/main/java/com/blockout/PaneParams.java

```diff
@@ -312,41 +412,49 @@ else if ((attr.startsWith("rgb(") || attr.startsWith("rgba(")) && m.find())
         }
     }
 
-    private static int getRGBA(String attr, Matcher m)
+    private static int getRGBA(final String attr, final Matcher m)
     {
-        int r = MathHelper.clamp_int(Integer.parseInt(m.group(1)), 0, 255);
-        int g = MathHelper.clamp_int(Integer.parseInt(m.group(2)), 0, 255);
-        int b = MathHelper.clamp_int(Integer.parseInt(m.group(3)), 0, 255);
+        final int r = MathHelper.clamp_int(Integer.parseInt(m.group(1)), 0, 255);
+        final int g = MathHelper.clamp_int(Integer.parseInt(m.group(2)), 0, 255);
+        final int b = MathHelper.clamp_int(Integer.parseInt(m.group(3)), 0, 255);
 
         int color = ((r & 0xFF) << 16) | ((g & 0xFF) << 8) | (b & 0xFF);
 
         if (attr.startsWith("rgba"))
         {
-            int alpha = (int) (Double.parseDouble(m.group(4)) * 255.0F);
+            final int alpha = (int) (Double.parseDouble(m.group(4)) * 255.0F);
             color |= MathHelper.clamp_int(alpha, 0, 255) << 24;
         }
 
         return color;
     }
 
-    private static int getColorByNumberOrName(int def, String attr)
+    private static int getColorByNumberOrName(final int def, final String attr)
     {
         try
         {
             return Integer.parseInt(attr);
         }
-        catch (NumberFormatException ex)
+        catch (final NumberFormatException ex)
         {
             return Color.getByName(attr, def);
         }
     }
 
+    /**
+     * Size pair of width and height.
+     */
     public static class SizePair
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this class from 26 to the maximum allowed 25 or externalize it in a public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2972)